src/mount.ts

```diff
@@ -133,6 +137,46 @@ export function mount(originalComponent: any, options?: MountingOptions) {
   const { emitMixin, events } = createEmitMixin()
   vm.mixin(emitMixin)
 
+  // stubs
+  if (options?.global?.stubs) {
+    transformVNodeArgs((args) => {
+      // regular HTML Element. Do not stubs these
+      if (Array.isArray(args) && typeof args[0] === 'string') {
+        return args
+      }
+
+      // don't care about comments/fragments
+      if (typeof args[0] === 'symbol') {
+        return args
+      }
+
+      // do not stub the VTU Parent component
+      if (typeof args[0] === 'object' && args[0]['name'] === 'VTU_COMPONENT') {
+        return args
+      }
+
+      if (
+        typeof args[0] === 'object' &&
+        args[0]['name'] in options?.global?.stubs
+      ) {
+        const name = args[0]['name']
+        // default stub
+        if (options?.global?.stubs[name] === true) {
```

Comment:
You are already in the conditional check, its there.

```suggestion
        if (options.global.stubs[name] === true) {
```

Reply:
heh, copy pasted... thanks, I'll clean this up